.github/workflows/release.yml

```diff
@@ -482,6 +482,33 @@ jobs:
           push: true
           tags: ethereumoptimism/deployer-bedrock:${{ needs.release.outputs.contracts-bedrock }},ethereumoptimism/deployer-bedrock:latest
 
+  balance-monitor:
```

Comment:
This looks okay but you might want to consider using the circleci [docker-build](https://github.com/ethereum-optimism/optimism/blob/develop/.circleci/config.yml#L93) and [docker-publish](https://github.com/ethereum-optimism/optimism/blob/develop/.circleci/config.yml#L149) jobs since they run on every push and are easier to test / dont require master release builds.

However if you are using changeset versioning, then the way you have it works as well.